### ISSUE TITLE
Not compatible with RN49 on iOS

### ISSUE THREAD

**Issue by alexmngn** [luggit/react-native-config#173]
I have an error trying to use this package with RN49. It was working fine with RN47. Here is the error I get from Xcode:

```
[15535:988556] Module ReactNativeConfig requires main queue setup since it overrides `constantsToExport` but doesn't implement `requiresMainQueueSetup. In a future release React Native will default to initializing all native modules on a background thread unless explicitly opted-out of.
```
The application runs but impossible to get anything from RN config. 

This returns an empty object:
```
import Config from 'react-native-config';
console.log(Config);
```

**Comment by alexmngn** [comment]
Actually this is fixed in https://github.com/luggit/react-native-config/pull/172

**Comment by pedro** [comment]
Oops, fixed in 0.9.0.

Thanks folks!
